custom_components/ble_monitor/const.py

```diff
@@ -588,6 +588,7 @@ class BLEMonitorBinarySensorEntityDescription(
 # - [instantly updating sensor list]:   sensors that update the state instantly after new data
 # - [binary sensor list]:               binary sensors
 MEASUREMENT_DICT = {
+    'b-parasite'              : [["temperature", "humidity", "moisture", "voltage", "rssi", "illuminance"], [], []],
```

Comment:
Move this line to the same position in the dictionary as you do in the next dictionary, where you have put this just below the Tempo Disc THPD sensor

Do not use the "brand", but use a name of the specific sensor, e.g. `Plant sensor v1`, as I wrote in an earlier comment.